packages/react-scripts/config/webpack.config.js

```diff
@@ -116,15 +116,15 @@ module.exports = function(webpackEnv) {
               stage: 3,
             }),
           ],
-          sourceMap: isEnvProduction ? shouldUseSourceMap : isEnvDevelopment,
```

Comment:
Is this change coming from the facebook/master ?

Reply:
Yup here is the PR for reference
https://github.com/facebook/create-react-app/pull/6472/files